pmsoService/materials/views.py

```
@@ -977,14 +972,25 @@ def get_material_price_statistics(request, material_id):
 
 class MaterialRequestListCreateView(generics.ListCreateAPIView):
-    queryset = MaterialRequest.objects.all()
     serializer_class = MaterialRequestSerializer
     filter_backends = [DjangoFilterBackend, OrderingFilter, SearchFilter]
     search_fields = ["request_name"]
     filterset_class = MaterialRequestFilter
     ordering_fields = ["is_urgent", "created_at", "due_date", "request_name"]
     pagination_class = MaterialRequestPagination
 
+    def get_queryset(self):
+        query = (MaterialRequest.objects
```

Ruff (0.14.0)

`MaterialRequest` may be undefined, or defined from star imports

(F405)